README.md

```diff
@@ -2,64 +2,107 @@
-  developers.
-- Infrastructure agnostic packaging of applications and their dependencies.
-- Scheduling applications into different clusters, zones, and regions.
-
-Crossplane does not:
-
-- Require that you run your workloads on Kubernetes.
-- Manage the data plane across Kubernetes clusters.
-- Manage or provision non-hosted Kubernetes clusters.
-
-Crossplane can be [installed](docs/getting-started/install.md) into any Kubernetes cluster, and
-is compatible with any Kubernetes-native project. It manages external services
-by installing [Custom Resource
-Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
-(CRDs) and
-[reconciling](https://kubernetes.io/docs/concepts/architecture/controller/)
-instances of those Custom Resources. Crossplane is built to be extensible,
-meaning that anyone can add functionality for an new or existing cloud provider.
-
-Crossplane is comprised of four main components:
-
-1. **Core Crossplane**: the set of Kubernetes CRDs and controllers that manage
-   installation of `providers`, `stacks`, and `applications`, as well as the
-   scheduling of configuration data to remote Kubernetes clusters.
-2. **Providers**: the set of Kubernetes CRDs and controllers that provision and
-   manage services on cloud providers. A cloud provider is any service that
-   exposes infrastructure via an API.
-    - Examples: [Google Cloud
-      Platform](https://github.com/crossplane/provider-gcp), [Amazon Web
-      Services](https://github.com/crossplane/provider-aws),
-      [Azure](https://github.com/crossplane/provider-azure),
-      [Alibaba](https://github.com/crossplane/provider-alibaba),
-      [Github](https://github.com/crossplane/provider-github)
-3. **Stacks**: a bundled set of custom resources that together represent an
-   environment on a cloud provider. The bundle of instances can be created by a
-   single custom resource.
-   - Examples: [GCP Sample
-     Stack](https://github.com/crossplane/stack-gcp-sample), [AWS Sample
-     Stack](https://github.com/crossplane/stack-aws-sample), [Azure Sample
-     Stack](https://github.com/crossplane/stack-azure-sample)
-4. **Applications**: a deployable unit of code and configuration, which, when
-   created, may involve provisioning new services which are managed by a
-   `provider`, or consuming services created by a `stack`.
-    - Examples: [Wordpress](https://github.com/crossplane/app-wordpress)
-
-The full vision and architecture of the Crossplane project is described in our
-[architecture document].
-
-For more information, take a look at the official Crossplane [documentation].
+Crossplane is an open source Kubernetes add-on that extends any cluster with
+the ability to provision and manage cloud infrastructure, services, and
+applications using kubectl, GitOps, or any tool that works with the Kubernetes
+API.
+
+With Crossplane you can:
+
+* **Provision & manage cloud infrastructure and services with kubectl**
```

Comment:
And I also feel it's weird to forcefully push 'app' in the middle of infra sentences. Can we define application and infrastructure as two sections and promote each other? (we can add `experimental `or `alpha` tag to app section if needed)

Reply:
Nit: This heading uses "&" in some places and "and" in others. 

```suggestion
* **Provision and manage cloud infrastructure and services with kubectl**
```

Reply:
> Can we define application and infrastructure as two sections and promote each other?
Yes, we have three sections that reference each other and progressively build to the app section.

> Nit: This heading uses "&" in some places and "and" in others.
Fixed.